one real build failure:
```
======================================================================
ERROR: test_convert_form_export_stock_in_repeat (corehq.apps.export.tests.test_export_conversion.TestConvertStockFormExport)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/vendor/lib/python2.7/site-packages/mock.py", line 1224, in patched
    return func(*args, **keywargs)
  File "/mnt/commcare-hq-ro/corehq/apps/export/tests/test_export_conversion.py", line 680, in test_convert_form_export_stock_in_repeat
    instance, _ = convert_saved_export_to_export_instance(self.domain, saved_export_schema)
  File "/mnt/commcare-hq-ro/corehq/apps/export/utils.py", line 228, in convert_saved_export_to_export_instance
    saved_export.save()
  File "/vendor/lib/python2.7/site-packages/couchdbkit/schema/base.py", line 134, in save
    db.save_doc(doc, **params)
  File "/vendor/lib/python2.7/site-packages/couchdbkit/client.py", line 516, in save_doc
    **params).json_body
  File "/vendor/lib/python2.7/site-packages/restkit/resource.py", line 153, in put
    headers=headers, params_dict=params_dict, **params)
  File "/vendor/lib/python2.7/site-packages/couchdbkit/resource.py", line 140, in request
    response=e.response)
ResourceConflict: Document update conflict.
```
but looks good to me. 

re:

> the more i'm looking at the migration log the more impossible this task seems. i think there are a few bugs in the export generation code in the legacy system that generate indexes that are impossible to map to the new exports and don't yield any data anyways

I think at some point we should just punt and fall back to the remote app-style thing. is that an option?

--- a/corehq/apps/export/tests/data/saved_export_schemas/stock_form_export_repeat.json
+++ b/corehq/apps/export/tests/data/saved_export_schemas/stock_form_export_repeat.json
@@ -1,6 +1,5 @@
 {
-    "_id": "6d7dfbcd373fdf08bb53f789e3ed4085",
-    "_rev": "5-77bce367d7a323b55a676acb0febe77b",
+    "_id": "6d7dfbcd373fdf08bb53f789e3ed4088",
     "app_id": "49b24e49b80a9ff27a7077df27b5d1b8",
     "converted_saved_export_id": null,
     "default_format": "xlsx",

